### PR TITLE
Update Azure documentation for SendGrid dotnet

### PR DESCRIPTION
**Pull request by laur-craciun**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@laur-craciun__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: articles/app-service-web/sendgrid-dotnet-how-to-send-email.md [hunk 1]
```diff
@@ -139,22 +143,35 @@ After creating an email message, you can send it using
 the Web API provided by SendGrid. Alternatively, you may [use .NET's built in library](https://sendgrid.com/docs/Code_Examples/csharp.html).
 
 Sending email requires that you supply your
-SendGrid account credentials (username and password). The following code
-demonstrates how to wrap your credentials in a **NetworkCredential**
-object:
+SendGrid account credentials (username and password) or SendGrid API Key. API Key is the preferred method. API Keys are in beta. To configure API keys, visit [https://sendgrid.com/beta/settings/api_keys](https://sendgrid.com/beta/settings/api_keys)
+
```
Comment (radusg) [inline articles/app-service-web/sendgrid-dotnet-how-to-send-email.md]: Replace https://sendgrid.com/beta/settings/api_keys with https://app.sendgrid.com/settings/api_keys

**Comment by davidwrede** [comment]
Hi @laur-craciun, I submitted a couple of minor revisions to your PR. There's a couple of clean up items and making sure the topic reflects the correct "Last Updated" date value.

If you're good with those, then please merge the changes and I'll then merge them into the azure-content master branch.

Thanks,
Dave

**Comment by davidwrede** [comment]
Also, please sign the CLA via the link provided in this conversation thread. I'll need that done before I can formally accept the changes.

Thanks,
Dave

**Comment by laur-craciun** [comment]
Hello @davidwrede,
I have closed this Pull request because I don't have the 'contribution license agreement' signed. But it has been reopened by another colleague here:
https://github.com/Azure/azure-content/pull/4753

**Pull request closed without merging**